packages/presentation/src/PresentationTool.tsx

```diff
@@ -173,11 +175,10 @@ export default function PresentationTool(props: {
   const navigate = useMemo(() => debounce<PresentationNavigate>(_navigate, 50), [_navigate])
 
   const [state, dispatch] = useReducer(presentationReducer, {}, presentationReducerInit)
-
-  const perspective = useMemo(
-    () => (params.perspective ? 'published' : 'previewDrafts'),
-    [params.perspective],
-  )
+  const {bundlesPerspective, perspective: globalPerspective = 'previewDrafts'} = usePerspective()
+  const perspective = (
+    globalPerspective.startsWith('bundle.') ? bundlesPerspective : globalPerspective
+  ) as PresentationPerspective
```

Comment:
Just a note: we will need to update this once https://github.com/sanity-io/sanity/pull/7936 is merged